package.json

```diff
@@ -17,8 +17,8 @@
         "prettier": "prettier --write src",
         "start": "npm-run-all -p start:dev watch:css watch:js",
         "prestart": "npm run clean-build-dir",
-        "start:dev": "PORT=3000 NODE_ENV=development nodemon --inspect=0.0.0.0:9222 src --config nodemon.config.json",
-        "start:prod": "NODE_ENV=production node src",
+        "start:dev": "INLINE_RUNTIME_CHUNK=false PORT=3000 NODE_ENV=development nodemon --inspect=0.0.0.0:9222 src --config nodemon.config.json",
+        "start:prod": "INLINE_RUNTIME_CHUNK=false NODE_ENV=production node src",
```

Comment:
Just curious: do we need to set INLINE_RUNTIME_CHUNK in staging & prod as well?

Reply:
It's ignored for dev env, will remove for dev. Staging and Prod are required for CSP.